Towny/src/main/java/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -3712,6 +3714,14 @@ public static void parseTownClaimCommand(Player player, String[] split) throws T
 			selection.add(key);
 			outpost = true;
 
+		} else if (split.length == 1 && "fill".equalsIgnoreCase(split[0])) {
+			checkPermOrThrow(player, PermissionNodes.TOWNY_COMMAND_TOWN_CLAIM_FILL.getNode());
+			
+			final BorderUtil.FloodfillResult result = BorderUtil.getFloodFillableCoords(town, key);
+			if (result.feedback() != null)
+				TownyMessaging.sendErrorMsg(player, result.feedback());

```

Comment:
Should this be a `throw new TownyException()` situation? It looks like we'll still be passing on a selection from the results.coords().